src/main/java/use_case/translateText/TranslateTextInteractor.java

```diff
@@ -0,0 +1,47 @@
+package use_case.translateText;
+
+import entity.TextTranslator;
+import interface_adapter.translateText.TranslateTextPresenter;
+
+/**
+ * The TranslateText Interactor.
+ */
+public class TranslateTextInteractor {
+
+    private final TranslateTextDataAccessInterface userDataAccessObject;
+    private final TranslateTextPresenter userPresenter;
+    private final TextTranslator textTranslator;
+
+    public TranslateTextInteractor(TranslateTextDataAccessInterface translateTextUserDataAccessInterface,
```

Comment:
could you rename that parameter translateTextUserDataAccessInterface to something like translateTextDataAccessInterface or translateTextDAI? (remove the User in that variable name since it is not relevant to our program)